exercises/crypto-square/canonical-data.json

```diff
@@ -1,82 +1,27 @@
 {
   "exercise": "crypto-square",
-  "version": "2.0.0",
+  "version": "3.0.0",
   "cases": [
     {
       "description": "the spaces and punctuation are removed from the English text and the message is downcased",
       "cases": [
```

Comment:
Should we still be using sub `cases` keys if we are only testing one property?

Reply:
Hmm, good question! I see [in the README](https://github.com/exercism/problem-specifications/blob/master/README.md#test-data-format-canonical-datajson):

> Test cases can be arbitrarily grouped with a description to make organization easier.

These `cases` all have to do with "normalizing" text, while the next group of `cases` deal with "chunking" and ordering letters. I'm not sure whether or not that "makes organization easier." I'm happy to dump the sub `cases` or save that re-organization for when we add new tests.

Reply:
I'm not sure what the answer is either, I'm OK with leaving it as it is for now as the logical separation is still there even if the property name has changed.
If it ends up breaking someone's test generator I'm sure they'll submit a PR to fix it.